GTest deprecated `TYPED_TEST_CASE`

Warning is as follows:
```
pcl/test/common/test_geometry.cpp:49:1: warning: 'TypedTestCaseIsDeprecated' is deprecated: TYPED_TEST_CASE is deprecated, please use TYPED_TEST_SUITE [-Wdeprecated-declarations]
TYPED_TEST_CASE(XYZPointTypesTest, XYZPointTypes);
^
```
This occurs in many places throughout tests.

## Your Environment
* Operating System and version: Linux
* Compiler: clang 8.0.1
* PCL Version: master

The GTest used is from the [master branch](https://github.com/google/googletest) as per the "gtest" best-practices instead of OS installed version.

## Expected Behavior
No warnings

## Possible Solution
A simple `sed` might solve this

--- a/apps/cloud_composer/src/cloud_view.cpp
+++ b/apps/cloud_composer/src/cloud_view.cpp
@@ -42,7 +42,8 @@ pcl::cloud_composer::CloudView::CloudView (ProjectModel* model, QWidget* parent)
 }
 
 pcl::cloud_composer::CloudView::CloudView (const CloudView& to_copy)
-  : vis_ (to_copy.vis_)
+  : QWidget()
+  , vis_ (to_copy.vis_)
   , model_ (to_copy.model_)
   , qvtk_ (to_copy.qvtk_)
 {

